raiden-dapp/src/components/GeneralMenu.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div class="general-screen-menu">
+    <v-row class="general-screen-menu__account-details no-gutter" no-gutters>
+      <v-col cols="12">
+        <div class="general-screen-menu__account-details__title">
```

Comment:
Just as general feedback: We use CSS BEM naming and I think the modifier part gets separated by `--` so it would be: `general-screen-menu__account-details--title`